Fit `Card` headline on single line (as much as possible)

robinandeer opened the pull request:
## What?

Group card components in same module.
Update Card stories with proper text components.
Unify breakpoints in card component.
Align arrow icon with paragraph text.
Make headline text slightly smaller on large screens.

## Why?

To fit the card headlines on a single line (for native market languages at least).

I grouped all the components together since they are now so linked in the layout.

<!-- Finally, if that makes sense, add screenshots and/or screen recordings below, preferably with headlines and/or descriptions -->

## Sweden

![Screenshot 2021-05-10 at 12 08 31](https://user-images.githubusercontent.com/1220232/117643875-2c7da600-b189-11eb-98aa-8b977b79ebb2.png)

![Screenshot 2021-05-10 at 12 08 24](https://user-images.githubusercontent.com/1220232/117643872-2be50f80-b189-11eb-9475-b23fa6631686.png)

## Norway

![Screenshot 2021-05-10 at 12 08 42](https://user-images.githubusercontent.com/1220232/117643909-369fa480-b189-11eb-937c-79497959ec8f.png)

![Screenshot 2021-05-10 at 12 08 47](https://user-images.githubusercontent.com/1220232/117643916-37383b00-b189-11eb-9f79-9ea034a211e4.png)

## Denmark

![Screenshot 2021-05-10 at 12 08 53](https://user-images.githubusercontent.com/1220232/117643974-46b78400-b189-11eb-8c47-203dacbe8b13.png)

![Screenshot 2021-05-10 at 12 09 02](https://user-images.githubusercontent.com/1220232/117643980-47501a80-b189-11eb-9b4d-36f9f0a230d8.png)

## Mobile

![Screenshot 2021-05-10 at 12 09 48](https://user-images.githubusercontent.com/1220232/117644006-4dde9200-b189-11eb-9cdb-95b3ad113b69.png)

## Tablet

![Screenshot 2021-05-10 at 12 10 09](https://user-images.githubusercontent.com/1220232/117644028-559e3680-b189-11eb-99b3-3dd23e6e1b04.png)

Commit:
Fit card title on single line

Group card components in same module.
Update Card stories with proper text components.
Unify breakpoints in card component.
Align arrow icon with paragraph text.

gustaveen commented on src/client/pages/Landing/components/Card.tsx:
```diff
@@ -76,7 +76,7 @@ const CardHeader = styled.div`
   }
 
-  @media (min-height: 620px) {
+  @media (min-height: 600px) {
```
Wasn't all `@media (min-height:` set to `620px` before? :) Or maybe this change makes sense, just wanted to check that you don't mix it up with the `min-width` media query that's set to `600px`

robinandeer replied:
Good call! Will change back

gustaveen commented on src/client/pages/Landing/components/Card.tsx:
```diff
@@ -128,11 +132,58 @@ const CardContainer: React.FC<{
 }
 
-export const Card: React.FC<{
+export const CardHeadline = styled.h2<{ disabled?: boolean }>`
```
Just for the sake of keeping the css before the react components, maybe put this above `CardContainer` :)

gustaveen commented on src/client/pages/Landing/components/Card.tsx:
```diff
@@ -128,11 +132,58 @@ const CardContainer: React.FC<{
+  }
+
+  @media (min-width: 1020px) {
+    font-size: 1.9rem;
```
This solves the layout issue for the DK case, just wondering if we should go outside our defined font-size? (You don't know about them of course so hard for you to know:D) https://www.figma.com/file/zlX4FcM5dMRz9stsCNQJXB/Web-UI-Kit?node-id=0%3A1 I created a review app so we can let Mange have a look.

gustaveen reviewed (commented):
Good job Robin! 🙌  Just some thoughts :)

Commit:
Group styled components in Card

Restore min-height media query.

Commit:
Make CardContainer responsible for Card margin

Remove edge-margin for first/last Card component.
Remove unused bottom margin from Card component.

robinandeer commented:
What do you think about the latest solution?

We keep the 32px font size but remove the edge-margin for the first/last Card. I also moved the responsibility to add margin between `Card` to the `CardContainer` -- how does that work in your mind?

## Updated screenshots

![Screenshot 2021-05-10 at 16 07 50](https://user-images.githubusercontent.com/1220232/117672791-7460f500-b1aa-11eb-852a-6be860c4ce2d.png)

![Screenshot 2021-05-10 at 16 07 57](https://user-images.githubusercontent.com/1220232/117672804-76c34f00-b1aa-11eb-8fcf-eb60fcc0ed50.png)

gustaveen reviewed (approved):
🙌
